surveyform.html

```diff
@@ -79,7 +79,8 @@ <h1 id="title">freeCodeCamp Survey Form</h1>
   <button type="button" value="submit" style="width:650px;height:40px;font-size:18px;color: rgb(241, 234, 234);border: none;background-color: rgb(47, 173, 79);">submit</button>
```

Comment:
Please try to avoid inline styling to ensure you follow html and css best practices as described in Microverse Inc